vertx-mongo-client/src/main/generated/io/vertx/rxjava/ext/mongo/MongoClient.java

```diff
@@ -849,6 +850,102 @@ public MongoClient dropCollection(String collection, Handler<AsyncResult<Void>>
   }
 
   /**
+   * Creates an index.
+   * @param collection the collection
+   * @param key A document that contains the field and value pairs where the field is the index key and the value describes the type of index for that field. For an ascending index on a field, specify a value of 1; for descending index, specify a value of -1.
+   * @param resultHandler will be called when complete
+   * @return 
+   */
+  public MongoClient createIndex(String collection, JsonObject key, Handler<AsyncResult<Void>> resultHandler) { 
+    delegate.createIndex(collection, key, resultHandler);
+    return this;
+  }
+
+  /**
+   * Creates an index.
+   * @param collection the collection
+   * @param key A document that contains the field and value pairs where the field is the index key and the value describes the type of index for that field. For an ascending index on a field, specify a value of 1; for descending index, specify a value of -1.
+   * @return 
+   */
+  public Observable<Void> createIndexObservable(String collection, JsonObject key) { 
```

Comment:
I don't understand you. This class generated from "parent" MongoClient.java automatically. I believe this is common rule for rxjava classes: first method with the same signature as in "parent" class and second one with `xObservable` postfix which returns a `Future`